sass/_theme_rst.sass

```diff
@@ -298,6 +298,15 @@
       @extend .fa-download
       &:before
         margin-right: 4px
+  .guilabel
+    border: 1px solid lighten($class-color, 25%)
+    background: lighten($class-color, 50%)
```

Comment:
This shouldn't piggy back off of `class-color` and should set it's own color, or use something more appropriate than `class-color`.